openapi/multi-yaml/index.yaml

```diff
@@ -79,223 +79,141 @@ tags:
-
-        ### [Payments API](http://localhost:3001/api-spec#tag/Payments/operation/CreatePayment) Example
-
-        ##### Example Request: Create Payment Using Wallet Token
-        ```json
-        {
-          "amount": 1000,
-          "currency": "usd",
-          "payment_method": {
-            "token": "pm_abc123xyz"
-          }
+      // Example: Create payment method group
+      const group = await fetch('https://api.justifi.ai/v1/payment_method_groups', {
+        method: 'POST',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': platformWalletAccountId,  // Platform wallet account
+          'Content-Type': 'application/json'
         }
-        ```
-
-      ##### Example Response: Payment with Auto Cloned Token
-      ```json
-      {
-        "id": "py_xyz789abc",
-        "type": "payment",
-        "data": {
-          "amount": 1000,
-          "currency": "usd",
-          "status": "succeeded",
-          "payment_method": {
-            "id": "pm_autoclonedxyz789",
-            "signature": "4bHXoVaoR",
-            "account_id": "acc_456subaccount",
-            "status": "valid",
-            "card": {
-              "id": "pm_autoclonedxyz789",
-              "name": "Lindsay Whalen",
-              "acct_last_four": "4242",
-              "brand": "visa",
-              "token": "pm_autoclonedxyz789",
-              "month": "5",
-              "year": "2025"
-            }
-          }
-        }
-      }
+      });
+
+      // Example: Add payment methods to group
+      const updatedGroup = await fetch(`https://api.justifi.ai/v1/payment_method_groups/${groupId}`, {
+        method: 'PATCH',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': platformWalletAccountId,  // Platform wallet account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "payment_method_ids": ["pm_walletaccxyz", "pm_walletaccabc"]
+        })
+      });
       ```
 
-      ### [Checkout Flow](http://localhost:3001/api-spec#tag/Checkouts/operation/CreateCheckout) Example
-
-      When using platform wallet payment methods with checkouts:
-      1. Create a checkout for the sub-account, specifying a payment method group from the wallet account
-      2. Complete the checkout using a payment method token from the wallet account
-
-      ##### Example Request: Create Checkout
-      ```json
-      {
-        "amount": 10000,
-        "description": "order_xyz",
-        "origin_url": "http://localhost:3000",
-        "payment_method_group_id": "pmg_xyz123",
-        "statement_descriptor": "Big Business"
-      }
-      ```
+      ### 2. Process Payments
+      You can process payments using wallet payment methods through either our component or API approach. All payment operations require the processing sub account ID in the Sub-Account header.
```

Comment:
I'd change this to '... either through the Unified Fintech Checkout web component (https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs) or via API.'